src/substrait/textplan/parser/Tool.cpp

```diff
@@ -0,0 +1,55 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <getopt.h>
+#include <iostream>
+
+#include "substrait/textplan/SymbolTablePrinter.h"
+#include "substrait/textplan/parser/ParseText.h"
+
+namespace io::substrait::textplan {
+namespace {
+
+void readText(const char* filename) {
+  auto stream = io::substrait::textplan::loadTextFile(filename);
+  if (!stream.has_value()) {
+    std::cerr << "An error occurred while reading: " << filename << std::endl;
```

Comment:
For future reference, should we introduce a macro or error logging mechanism? Definitely not for this PR, but for future work. Any thoughts?

Reply:
The parser and converters have the capability to handle multiple errors long with their source locations.  This particular file is a special case similar only to the converter's Tool.cpp.  Eventually these two will be merged and there will be only one place handling errors like this (everything else is going to rely on the caller to deal with the list of error messages).